test/test.construct.ts

```diff
@@ -505,3 +517,23 @@ class MyBeautifulConstruct extends Construct {
     super(scope, id);
   }
 }
+
+class MyAlmostBeautifulConstruct extends Construct {
+  public status: string = 'PrePrepared'; 
+
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+  }
+
+  protected onPrepare() {
+    this.status = 'Prepared'
+  }
+}
+
+class MyBeautifulAspect implements IAspect {
```

Comment:
Minor: I would use a name for this class to signify that it is adding a new construct to the construct tree, something like `AddAnotherConstructAspect` or something like that.

Reply:
done